lib/core/src/Cardano/Pool/Ranking.hs

```diff
@@ -104,11 +106,46 @@ desirability
     -> Double
 desirability constants pool
     | f_saturated <= c = 0
-    | otherwise    = (f_saturated - c) * (1 - m)
+    | otherwise        = (f_saturated - c) * (1 - m)
   where
     f_saturated = saturatedPoolRewards constants pool
     m = getRatio $ margin pool
-    c = fromIntegral $ getLovelace $ cost pool
+    c = fromIntegral $ getQuantity $ cost pool
+
+-- | The saturation-level of a pool indicate how far a pool is from the
+-- desirated relative stake fixed by the network. A saturation level above 1
+-- means that the pool is satured. A level of 0.5 / 50% means that the pool owns
```

Comment:
> is satured

Wouldn't

```
== 1 -> saturated
> 1 -> oversaturated
```

be more logical?

Reply:
satured means that the pool as too much stake. "oversaturated" isn't really a thing. A pool is saturated by 1%, 100% etc .. but it's just .. saturated.